conf/quincy/cephadm/tier-0.yaml

```diff
@@ -0,0 +1,46 @@
+# System Under Test environment configuration for Tier-0 test suites for build qualification.
```

Comment:
IMO, we can reuse `minimal.yaml` in `conf` folder.

Reply:
@rkhambad could you let me why we are creating a separate yaml file?

Reply:
For now... we can leave it. If we go to the root folder, custom test execution would break.